### PR TITLE
Add validate tool

### PR DESCRIPTION
**Pull request by wallyqs**

### CODE REVIEW

File: cmd/nats-rest-config-validator/README.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# NATS REST Configuration Validator
+
+Validate your config without running the server.
```
Comment (ColinSullivan1) [inline cmd/nats-rest-config-validator/README.md]: Validate your configuration from the command line without...

File: cmd/nats-rest-config-validator/main.go [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+		fmt.Fprintln(os.Stderr, usageStr)
+	}
+
+	opts := &server.Options{
+		NoLog: true,
+	}
+	flag.StringVar(&opts.DataDir, "data", ".", "Directory for storing data.")
+	flag.StringVar(&opts.DataDir, "dir", ".", "Directory for storing data.")
+	flag.StringVar(&opts.DataDir, "d", ".", "Directory for storing data.")
+	flag.Parse()
```
Comment (ColinSullivan1) [inline cmd/nats-rest-config-validator/main.go]: Need to add a `-v`?

File: cmd/nats-rest-config-validator/main.go [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+	"math/rand"
+	"os"
+	"time"
+
+	"github.com/nats-io/nats-rest-config-proxy/internal/server"
+)
+
+const usageStr = `
+Options:
+    -d, --dir <directory>         Directory for storing data
```
Comment (ColinSullivan1) [inline cmd/nats-rest-config-validator/main.go]: Default is the current directory.

File: cmd/nats-rest-config-validator/README.md (hunk 1 above)
Comment (wallyqs) [inline cmd/nats-rest-config-validator/README.md]:
```suggestion
Validate your configuration from the command line without running the server.
```

File: cmd/nats-rest-config-validator/main.go (hunk 3 above)
Comment (wallyqs) [inline cmd/nats-rest-config-validator/main.go]:
```suggestion
    -d, --dir <directory>         Directory for storing data (default is the current directory.)
```

File: cmd/nats-rest-config-validator/main.go (hunk 2 above)
Comment (wallyqs) [reply cmd/nats-rest-config-validator/main.go]: for version right...

**Pull request merged**